### DISCUSSION TITLE
Icon Funnel - El icono no existe en Figma

### DISCUSSION THREAD

**Discussion by BrunoBiagioniNeto** [Telefonica/mistica-design#1429] [Q&A]
Este ícono no existe, no lo encuentro, es una actualización o está en otro lugar?

![image](https://github.com/Telefonica/mistica-design/assets/123083886/42814c75-2609-413a-a067-994b3b7c8cdd)

**Comment by yceballost** [comment]
Hi Bruno! 

Funnel icon only exists in O2 brand
![image](https://github.com/Telefonica/mistica-design/assets/6722153/95d89a16-08a7-4285-a824-c5c8378cb1f5)
https://github.com/Telefonica/mistica-icons?tab=readme-ov-file#funnel-regular

You can find it in O2 Brand Library
https://www.figma.com/file/wHTqJ7KDhGKrNSNpmMb9nW/O2?type=design

Anyway, in code, when you use an icon that does not exist in other brands, like this icon in Vivo (for example), the icon that will be shown is the icon that exists in other brands (Do you understand me? xD)
https://tinyurl.com/2cgkdj73

**Comment by AnaMontes11** [comment]
Hi @BrunoBiagioniNeto !
We are going to include the funnel icon proposed by @LucasAlbuquerque0 in the issue: New chip esim and funnel icons #1441
